README.md

```diff
@@ -30,6 +31,11 @@ Run in the R console:
 install.packages("pak")
 pak::pak("aim-rsf/mapmetadata")
 ```
+or 
```

Comment:
do we need 'or' or shall we just delete lines 31 and 32? Line 32 also says _aim-rsf_

Reply:
I deleted it - I think it's simpler to have one suggested installation option